tests/test_monkeypatch.py

```diff
@@ -42,7 +42,7 @@ def test_get_edit_link(self, mock_is_obj_review_locked):
         )
         # We test that moderation check is called when getting an edit link
         self.assertTrue(mock_is_obj_review_locked.called)
-        if versioning_version < "2":
+        if versioning_version != "2.0.0":
             # Edit link is inactive as `mock_is_obj_review_locked` is True
             self.assertIn("inactive", edit_link)
         else:
```

Comment:
```suggestion
        self.assertIn("inactive", edit_link)
```

**plus** deleting the next three lines